centreon/packages/ui/src/Form/Inputs/ConnectedAutocomplete.tsx

```diff
@@ -109,6 +116,7 @@ const ConnectedAutocomplete = ({
         disabled={disabled}
         error={error}
         field={filterKey}
+        filterOptions={getUniqueOptions}
```

Comment:
Use the renderUniq... From the utils file

Reply:
waiting to release-23.10-next to be merged on develop, to apply the feedback